sdk/resourcemanager/azure-resourcemanager-appplatform/src/main/java/com/azure/resourcemanager/appplatform/models/UserSourceType.java

```diff
@@ -19,6 +19,9 @@ public final class UserSourceType extends ExpandableStringEnum<UserSourceType> {
     /** Static value Source for UserSourceType. */
     public static final UserSourceType SOURCE = fromString("Source");
 
+    /** Static value BuildResult for UserSourceType. */
+    public static final UserSourceType BUILD_RESULT = fromString("BuildResult");
```

Comment:
Is this class now hand-written or codegen? Again relate to copyright.

Reply:
It's hand-written now.

Reply:
Eh, it is generated (you have `--preserve-model=UserSourceType,RuntimeVersion`)?